Session-6/task5/main.go

```diff
@@ -31,5 +31,6 @@ func main() {
 	}
 
 	//fmt.Printf("%d", addMap["go"])
+	//task 5 is done
```

Comment:
Bu cür daha qısa və effektiv üsul var. map["go"] kimi yaransa əgər (məsələn) o 1 dəfə yaranır və hər dəfə edit edir. Ona görə xüsusi yoxlanışlar və overengineering etməyə ehtiyac yoxdur

```suggestion
func main() {

	languages := []string{"go", "java", "go", "python", "go", "java"}
	addMap := map[string]int{}

	for _, v := range languages {
		addMap[v] += 1
	}

	fmt.Println(addMap)

}
```